### ISSUE TITLE
Design: Podify action button is hidden until you select a container

### ISSUE THREAD

**Issue by mairin** [containers/podman-desktop#2644]
TLDR; I think because the podify button is hidden until you make a selection, the capability is a bit hidden. It would help with visibility if the podify button was always visible - it could be greyed out until an active selection is made and it could be used, but I think if it's visible and on the screen it could help users be more aware of the feature.

## The issue

This is what it looks like on the Containers screen when no container is selected:
![Image](https://user-images.githubusercontent.com/799683/240970958-079f11e4-b2e4-406f-8534-79fd6fd9c3e0.png)

This is what it looks like when >= 1 container is selected:
![Image](https://user-images.githubusercontent.com/799683/240971069-37cab283-c802-4c99-8326-d9504ba76875.png)
